Moments Recorder/Scripts/Editor/RecorderEditor.cs

```diff
@@ -45,6 +46,7 @@ void OnEnable()
 			m_Width = serializedObject.FindProperty("m_Width");
 			m_Height = serializedObject.FindProperty("m_Height");
 			m_FramePerSecond = serializedObject.FindProperty("m_FramePerSecond");
+			m_FramesPerColorSample = serializedObject.FindProperty("m_FramesPerColorSample");
```

Comment:
`Recorder` doesn't have a serialized `m_FramesPerColorSample` field. Did you miss a file in your commit?

Reply:
You're quite right! I hadn't copied my changes to Recorder.cs across. It should be good to go now. Thanks, @Chman.